tests/test_main_cli_flags.py

```
@@ -0,0 +1,344 @@
+
+
+# ===========================================================================
+# Subtask 6: Executor/manager quiet mode
+# ===========================================================================
+
+class TestQuietMode:
+    def test_executor_quiet_suppresses_init_prints(self, capsys):
+        """Executor with quiet=True should not print extension messages."""
+        from ai_content_pipeline.pipeline.executor import ChainExecutor
+        fm = MagicMock()
+        executor = ChainExecutor(fm, quiet=True)
```

Ruff (0.14.14)

[error] Local variable `executor` is assigned to but never used

Remove assignment to unused variable `executor`

(F841)